src/shared/lib/MutationUtils.tsx

```diff
@@ -52,6 +57,18 @@ export function getProteinImpactTypeOptionDisplayValueMap(proteinImpactTypeColor
     [proteinImpactType: string]: string;
 }): { [proteinImpactType: string]: JSX.Element } {
     return {
+        [ProteinImpactType.MISSENSE]: (
```

Comment:
i can you see you didn't start this code but it's super repetitive. easy refactor?


```